test/output-files.js

```diff
@@ -59,15 +60,24 @@ describe('output-files', () => {
     coverageInfo[1].url.should.include('puppeteerTemp-inline-1.js')
   })
 
+  it('appropriately handles es6 modules', () => {
```

Comment:
perhaps rather than, `appropriately handles es6` modules, we could call this:

`appropriately handles modules required via http/https`?

likewise, let's call the fixture, `http-es6-modules.json`.

Reply:
Done